bundles/tampere/content-editor/view/SideContentEditor.js

```diff
@@ -298,6 +305,82 @@ Oskari.clazz.define('Oskari.tampere.bundle.content-editor.view.SideContentEditor
             me.currentEditFeatureFid = fid;
         },
 
+        /**
+         * Edit multiple feautures
+         * @method  _editMultipleFeatures
+         * @param   {String}              fid feature identifier
+         * @private
+         */
+        _editMultipleFeatures: function (fid) {
+            var me = this;
+            var geometry = me._findGeometryByFid(fid);
+            if (geometry != null) {
+                me.layerGeometries = geometry;
+            }
+            me.setGeometryType(me.layerGeometries.geometry.id);
+            jQuery.each(me.templateFeatureMarkup[0].rows || [], function(index, el) {
+                var element = jQuery(el);
+                jQuery.each(el.cells ||[], function(i, cell) {
+                    var cellEl = jQuery(cell);
+                    if(i > 0) {
+                        var input = cellEl.find('input');
+                        if(input.length > 0) {
+                            input.addClass('template-feature');
+                            input.val('');
+                            input.attr('data-default', '');
+                            input.off('blur.modifyMultipleFeatures');
+                        } else {
+                            cellEl.text('');
+                        }
+                    } else if(cellEl.text() === '__fid') {
+                        element.hide();
+                    }
+                });
+            });
+            var saveMultipleFeaturesButton = Oskari.clazz.create('Oskari.userinterface.component.Button');
+            saveMultipleFeaturesButton.setTitle(me.loc.buttons.save);
+            saveMultipleFeaturesButton.setHandler(function () {
+                var thisInput = jQuery(this);
+                var yesButton = Oskari.clazz.create('Oskari.userinterface.component.Button');
+                yesButton.setTitle(me.loc.buttons.yes);
+                yesButton.setHandler(function () {
+                    var templateInputs = jQuery('input.template-feature');
```

Comment:
This is gathering data from forms? Maybe create a method for it that can be tested?